orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -406,7 +406,8 @@ def _get_exported_model(self, input_schema, *inputs, **kwargs):
                                           'dynamic_axes': self._input_info.dynamic_axes,
                                           'verbose': self._debug_options.logging.log_level < LogLevel.WARNING,
                                           'export_params': False,
-                                          'keep_initializers_as_inputs': True}
+                                          'keep_initializers_as_inputs': True,
+                                          'operator_export_type': torch.onnx.OperatorExportTypes.ONNX_ATEN_FALLBACK}
```

Comment:
Is this supported in pytorch 1.11 or requires nightly?

Reply:
What will happen if we use pytorch 1.11? 
1. This option will not be recognized, and all export will fail 
2. Export will fail only if aten fallback is required

Reply:
2.